src/stats.js

```diff
@@ -101,7 +101,7 @@ module.exports = (common) => {
         return
       }
 
-      ipfs.stats.bw().then((res) => {
+      return ipfs.stats.bw().then((res) => {
```

Comment:
@hacdias have you checked if this is done in all the tests?

Reply:
Yes, on all promises tests

Reply:
thank you 👍